### ISSUE TITLE
Building blenderpy on Fedora 32 with Python 3.8.5 fails

### ISSUE THREAD

**Issue by reddraconi** [TylerGubala/blenderpy#57]
## TL;DR:
I'm running a fresh install of Fedora 32 Workstation with Python 3.8.5.
Pulling down a copy of your repo, and attempting to build fails.

## To Reproduce
1. Clone the blenderpy repo to a local directory (~/blenderpy)
2. `cd` to `~/blenderpy`
3. Run `python -m pip install --upgrade pip wheel future-fstrings `
4. Run `python -m pip install -r requirements.txt`
5. Run `python ./setup.py sdist bdist_wheel`
6. Build fails with `error: [Errno 2] No such file or directory: 'build_linux_bpy/bin'`

## Expected Behavior
A complete build of compiled bpy libs in `dist`.

### Environment
**Desktop**
 - OS: Fedora 32 Workstation
 - Python: Python 3.8.5

**Additional context**
I'm trying to build from source, as trying to install bpy via pip doesn't work, either, as it reports
`Exception: 2.82 bpy is not compatible with Linux Python 3.8.5` during compilation.
Is this expected behavior?

**Comment by Mr-Steal-Your-Script** [comment]
i am having the same problem. i found this issue: https://github.com/TylerGubala/blenderpy/issues/24 from the past

**Comment by TylerGubala** [comment]
```
>>> import sys
>>> sys.version
'3.7.7 (default, Jun 13 2020, 11:11:23) [MSC v.1916 64 bit (AMD64)]'
```

Blender is built against Python version 3.7; you cannot build against Python 3.8 without making a custom build that I cannot personally maintain.

I recommend using Python 3.7

**Comment by reddraconi** [comment]
Understood. Thanks!
